app/views/hyrax/datasets/_work_files.html.erb

```diff
@@ -0,0 +1,6 @@
+<dl class="work-files">
```

Comment:
@coblej - what would you think about using some different markup here? I think it looks a little strange as a definition list and reads strangely with the word 'files' showing twice. Maybe just wrap it in a div with a unique class, change 'Files' to something like 'Size' and just use a separator (like an em-dash) to offset the count/size info? So something like: Size -- 2 files (300 Bytes)

Reply:
@coblej I agree with @mikedaul on the presentation: might benefit from a minor tweak here. I think it'd look better if it didn't mimic the style used for the citation.